ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -69,6 +70,23 @@ static int gcp_object_list_free(WT_FILE_SYSTEM *, WT_SESSION *, char **, uint32_
   __attribute__((__unused__));
 static int gcp_terminate(WT_STORAGE_SOURCE *, WT_SESSION *) __attribute__((__unused__));
 
+// Construct a pathname from the directory and the object name.
+static std::string
+gcp_path(const std::string &dir, const std::string &name)
```

Comment:
hmm name here feels too ambiguous, it feels like name variable is more of a path, is that correct?

Reply:
that's correct, thanks for picking this up, I'll change it now :)